src/gui/modalMenu.cpp

```diff
@@ -306,20 +316,24 @@ bool GUIModalMenu::preprocessEvent(const SEvent &event)
 			return true;
 		}
 	}
-#endif
 
 	if (event.EventType == EET_MOUSE_INPUT_EVENT) {
-		s32 x = event.MouseInput.X;
-		s32 y = event.MouseInput.Y;
+		if (!m_simulated_mouse) {
```

Comment:
it's fine now but in the future I think it'd be better to have this as a property of the `event`

Reply:
would require Irrlicht changes, so I'd prefer not to do it in this PR